test/second_q/algorithms/excited_state_solvers/test_excited_states_solvers.py

```diff
@@ -185,6 +186,42 @@ def custom_filter_criterion(eigenstate, eigenvalue, aux_values):
 
         self._assert_energies(computed_energies, ref_energies, places=3)
 
+    @unittest.skipIf(not _optionals.HAS_PYSCF, "pyscf not available.")
+    def test_solver_compatibility_with_mappers(self):
+        """Test that solvers can use both QubitConverter and QubitMapper"""
+        with self.subTest("Excited states solver"):
+            # pylint: disable=unused-argument
+            def filter_criterion(eigenstate, eigenvalue, aux_values):
+                return np.isclose(aux_values["ParticleNumber"][0], 2.0)
+
+            # with qubit converter
```

Comment:
These cases should be different subtests such that it is easier to identify what test fails.
The same applies elsewhere.

Reply:
ok